distribution/openhabhome/configurations/scripts/jsr223_demo.js

```diff
@@ -0,0 +1,166 @@
+		var action = oh.getActions() ;
+		oh.logInfo("execute:"+__LINE__, "available actions: " + action.keySet());
+		var ping = oh.getAction("Ping");
+		oh.logInfo("execute:"+__LINE__, "ping: " + ping);
+		oh.logInfo("execute:"+__LINE__, "internet reachable: " + (ping.static.checkVitality("google.com", 80, 100) ? "yes" : "no"));
+		var Heating_FF_Child = ItemRegistry.getItem('Heating_FF_Child');
+		var Heating_GF_Corridor = ItemRegistry.getItem('Heating_GF_Corridor');
+        oh.logInfo("execute::item:"+__LINE__, Heating_FF_Child);
+        oh.logInfo("execute::DateTime:"+__LINE__, DateTime.now().minusMinutes(30));
+        oh.logInfo("execute::PersistenceExtensions :"+__LINE__, PersistenceExtensions.changedSince(Heating_FF_Child,DateTime.now().minusMinutes(600)));
+        //PersistenceExtensions.persist(Heating_GF_Corridor);
+        BusEvent.postUpdate("Heating_GF_Corridor", "ON");
+        //BusEvent.postUpdate(Heating_GF_Corridor, "OFF");
+        //BusEvent.sendCommand("Heating_GF_Corridor", "OFF");
+        oh.logInfo("execute::item:"+__LINE__, (Heating_GF_Corridor.state == OnOffType.OFF));
+	}
+};
+
+
+var ActionTest = new Rule(){
+    getEventTrigger: function(){
+        oh.logDebug("getEventTrigger", "self:"+this);
+        return [
+            new TimerTrigger("0/20 * * * * ?")
+        ];
+    },
+    execute: function(event){
+        oh.logInfo("ActionTest:"+__LINE__,"execute");
+
+        var action = oh.getActions();
+        var actionArr = Java.from(action.keySet());
+        oh.logInfo("execute:"+__LINE__, "available actions: " + action.keySet());//Exec, Transformation, Mail, Ping, HTTP, Audio, ...
+
+        for (var i=0; i<actionArr.length; i++) {
+             oh.logInfo("execute:"+__LINE__, "Action: " +  oh.getAction(actionArr[i]));
+        }
+
+    }
+};
+
+var FileOperations = new Rule(){
+    getEventTrigger: function(){
+        return [
+            new TimerTrigger("0/60 * * * * ?")
+        ];
+    },
+    execute: function(event){
+        oh.logInfo("FileOperations:"+__LINE__,"execute");
+
+        ////////////////////
+        // take Scriptfiles Array
+        //only lists Filenames from Folder
+        //var ohScriptFiles = transform("EXEC", "ls configurations/scripts","").split("\n");
+        //OR BETTER
+        //lists Filenames with its paths
+        var ohScriptFiles = FileUtils.listFiles(new File("configurations/scripts"), ["js"], false );
+        oh.logInfo("ohScriptFiles:"+__LINE__,ohScriptFiles);
+
+        ////////////////////
+        // read ScriptFile
+        //var scriptfileContent = transform("EXEC", "cat configurations/scripts/templateRule.js","")+"";
+        //OR BETTER
+        var scriptfileContent = FileUtils.readFileToString(new File("configurations/scripts/jsr223_demo.js"));
+        oh.logInfo("scriptfileContent:"+__LINE__, ohScriptFiles);
+
+    }
+};
+
+
+
+function getRules(){
+	return new RuleSet([TestRule, ActionTest, FileOperations]);
+}
+
```

Comment:
please remove this sample file and rather add it to the AddOn itself or (even better) enhance the Wiki page https://github.com/openhab/openhab/wiki/Jsr223-Script-Engine